app/src/main/java/com/yeonae/chamelezone/DialogFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.yeonae.chamelezone
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.util.DisplayMetrics
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import kotlinx.android.synthetic.main.fragment_dialog.*
+
+class DialogFragment : DialogFragment() {
```

Comment:
```suggestion
class YOUR_DialogFragment : DialogFragment() {
```
적절하게 이름을 변경해주세요. EXTENDS 하는 클래스랑 이름이 같아요

Reply:
수정했습니당